python/experimental/eminem_importer.py

```
@@ -0,0 +1,483 @@
+
+    if not input_path.exists():
+        raise FileNotFoundError(f"Input file not found: {input_file}")
+
+    if verbose:
+        print(f"Loading data from: {input_file}")
+
+    try:
+        # Try gzip first (files may be gzipped regardless of extension)
+        try:
+            with gzip.open(input_file, "rb") as f:
+                data = pickle.loads(f.read())
```

Ruff (0.15.15)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)